torch/ao/quantization/quantize_fx.py

```diff
@@ -178,6 +178,8 @@ def _prepare_fx(
     model: torch.nn.Module,
     qconfig_dict: Any,
     is_qat: bool,
+    sample_args: Optional[Tuple[Any]] = None,
```

Comment:
thoughts on trying to match the naming of `torch.jit.trace`'s `example_inputs`? (https://pytorch.org/docs/stable/generated/torch.jit.trace.html)

also, that API doesn't have kwargs, do we know why?